zio-http-gen/src/test/scala/zio/http/gen/scala/CodeGenSpec.scala

```diff
@@ -262,5 +325,5 @@ object CodeGenSpec extends ZIOSpecDefault {
           "/GeneratedUserNameArray.scala",
         )
       },
-    ) @@ java11OrNewer @@ flaky @@ blocking // Downloading scalafmt on CI is flaky
+    ) @@ java11OrNewer /*@@ flaky*/ @@ blocking // Downloading scalafmt on CI is flaky
```

Comment:
Note to self: revert this - flaky was removed just for local runs to not be executed 100 times on the failing test I added…